### PR TITLE
Changes related to multihomed and dualstack fairness.

### PR DESCRIPTION
**Pull request by palerikm**

### CODE REVIEW

File: draft-ietf-ice-rfc5245bis.xml [hunk 1]
```diff
@@ -1220,66 +1217,51 @@ between 1 and 256 inclusive.
...
-by a provider. If these concerns are important, the type preference
-for relayed candidates SHOULD be lower than host candidates. The
+<t>One criterion for selection of the type and local preference values
+is the use of a media intermediary, such as a TURN server, a tunnel
+service such as VPN server, or NAT.  With a media intermediary, if
+media is sent to that candidate, it will first transit the media
+intermediary before being received.  Relayed candidates are one type
+of candidate that involves a media intermediary.  Another are host
+candidates obtained from a VPN interface.  When media is transited
+through a media intermediary, it can have an positive or negative
```
Comment (akeranen) [inline draft-ietf-ice-rfc5245bis.xml]: nit: s/ an / a /

File: draft-ietf-ice-rfc5245bis.xml [hunk 2]
```diff
@@ -1200,14 +1200,11 @@ address, this value SHOULD be set to 65535. More generally, if there
...
 host is multihomed because it is dual-stack, the local preference
-SHOULD be set equal to the precedence value for IP addresses described
-in RFC 6724 <xref target="RFC6724"/>. If the host operating system
-provides an API for discovering preference among different addresses,
-those preferences SHOULD be used for the local preference to
-prioritize addresses indicated as preferred by the operating system.
-</t>
+should be set according to the current best practice described in <xref
+target="I-D.ietf-ice-dualstack-fairness"> ICE Multihomed and IPv4/IPv6
+Dual Stack Fairness </xref>.</t>
```
Comment (akeranen) [inline draft-ietf-ice-rfc5245bis.xml]: nit: s/ICE Multihomed and IPv4/IPv6 Dual Stack Fairness/"ICE Multihomed and IPv4/IPv6 Dual Stack Fairness" document/
Or drop the draft name; usually they are not mentioned in this kind of context.

File: draft-ietf-ice-rfc5245bis.xml [hunk 3]
```diff
@@ -1220,66 +1217,51 @@ between 1 and 256 inclusive.
...
 relayed candidates.</t>
 
-<t> Furthermore, if an agent is multihomed and has multiple IP
-addresses, the local preference for host candidates from a VPN
-interface SHOULD have a priority of 0. If multiple TURN servers are
-used, local priorities for the candidates obtained from the TURN
+<t>Furthermore, if an agent is multihomed and has multiple IP
+addresses, the recommendation in <xref target=
+"I-D.ietf-ice-dualstack-fairness"> ICE Multihomed and IPv4/IPv6 Dual
+Stack Fairness </xref> should be followed.  If multiple TURN servers
```
Comment (akeranen) [inline draft-ietf-ice-rfc5245bis.xml]: See comment above.

File: draft-ietf-ice-rfc5245bis.xml [hunk 4]
```diff
@@ -1220,66 +1217,51 @@ between 1 and 256 inclusive.
...
-</t>
+one.</t>
+
+<t>Another criterion for selection of preferences is IP address family.
+ICE works with both IPv4 and IPv6.  It therefore provides a transition
+mechanism that allows dual-stack hosts to prefer connectivity over
+IPv6, but to fall back to IPv4 in case the v6 networks are
+disconnected. Implementation should follow the guidelines from <xref target=
+"I-D.ietf-ice-dualstack-fairness"> ICE Multihomed and IPv4/IPv6 Dual
+Stack Fairness </xref> to avoid excessive delays in the connectivity check
```
Comment (akeranen) [inline draft-ietf-ice-rfc5245bis.xml]: Same here. I'd lean towards not mentioning draft name.

File: draft-ietf-ice-rfc5245bis.xml [hunk 5]
```diff
@@ -1220,66 +1217,51 @@ between 1 and 256 inclusive.
...
-</t>
-
-<t>
-Another criterion for selecting preferences is topological
-awareness. This is most useful for candidates that make use of
-intermediaries. In those cases, if an agent has preconfigured or
-dynamically discovered knowledge of the topological proximity of the
-intermediaries to itself, it can use that to assign higher local
-preferences to candidates obtained from closer intermediaries.
-</t>
```
Comment (akeranen) [inline draft-ietf-ice-rfc5245bis.xml:1282]: Why this paragraph is dropped? Seems still valid to me.

File: draft-ietf-ice-rfc5245bis.xml (hunk 1 above)
Comment (palerikm) [inline draft-ietf-ice-rfc5245bis.xml]: ok

File: draft-ietf-ice-rfc5245bis.xml (hunk 2 above)
Comment (palerikm) [inline draft-ietf-ice-rfc5245bis.xml]: Ok. Removed draft name

File: draft-ietf-ice-rfc5245bis.xml (hunk 3 above)
Comment (palerikm) [inline draft-ietf-ice-rfc5245bis.xml]: Ok

File: draft-ietf-ice-rfc5245bis.xml (hunk 4 above)
Comment (palerikm) [inline draft-ietf-ice-rfc5245bis.xml]: Agree.

File: draft-ietf-ice-rfc5245bis.xml (hunk 5 above)
Comment (palerikm) [inline draft-ietf-ice-rfc5245bis.xml:1282]: Added back.

**Comment by akeranen** [comment]
Looks good to me.

**Comment by pthatcherg** [comment]
LGTM (even though it's already merged :)

**Pull request merged**